pkg/codegen/go/gen_program.go

```diff
@@ -605,6 +605,8 @@ func (g *generator) genLocalVariable(w io.Writer, v *hcl2.LocalVariable) {
 			g.Fgenf(w, "if err != nil {\n")
 			g.Fgenf(w, "return err\n")
 			g.Fgenf(w, "}\n")
+		default:
```

Comment:
I had to add this. Without it the non-invoke function calls never make it into the output go file

Reply:
🤔 we are losing a number of go examples and I'm wondering if it's because of this change causing invalid go code. perhaps I should narrow this down.